disttask/framework/dispatcher/dispatcher_test.go

```diff
@@ -151,9 +151,8 @@ func TestSimple(t *testing.T) {
 	cnt := 40
 	taskMap := make(map[int64]*proto.Task)
 	for i := 0; i < cnt; i++ {
-		taskMap = dsp.(dispatcher.DispatcherForTest).GetRunningGlobalTasks()
-		if len(taskMap) == 1 {
-			break
+		if dsp.(dispatcher.DispatcherForTest).RunningGlobalTasksCnt() == 1 {
+			taskMap = dsp.(dispatcher.DispatcherForTest).GetRunningGlobalTasks()
```

Comment:
I think there will also be a data race in line159 or line165.
The current code does not support parallelism. I will fix https://github.com/pingcap/tidb/issues/42383. Or do you want to change it?

Reply:
If you fix it in your PR, you can try to enable the race for verifying it.

Reply:
Please close the issue after your pr merging.

Reply:
OK.